components/kyma-operator/pkg/kymahelm/kyma-helm-client.go

```diff
@@ -26,19 +28,21 @@ type ClientInterface interface {
 
 // Client .
 type Client struct {
-	helm *helm.Client
+	helm            *helm.Client
+	overridesLogger *logrus.Logger
 }
 
 // NewClient .
-func NewClient(host string, TLSKey string, TLSCert string, TLSInsecureSkipVerify bool) (*Client, error) {
+func NewClient(host string, TLSKey string, TLSCert string, TLSInsecureSkipVerify bool, logger *logrus.Logger) (*Client, error) {
```

Comment:
rename to overridesLogger. It's a good name :)

Reply:
done